packages/ts/react-form/test/useForm.spec.tsx

```diff
@@ -442,19 +442,23 @@ describe('@vaadin/hilla-react-form', () => {
     });
 
     it('should add item to empty array', async () => {
+      let val: Player[] | undefined;
+      let setVal: ((value: Player[] | undefined) => void) | undefined;
       const { findByTestId } = render(
         <TeamForm
           testAction={({ read, value, setValue }) => {
```

Comment:
BTW, instead of `read` here you could probably use `setValue` for the array only. I guess, it'll make SonarQube happy

Reply:
I try to do something vaguely similar to what would happen in a real application. Honestly, I don't care that much about code repetition in tests: in some cases, it's even a benefit as it makes a single test easier to read.

Reply:
Ok, but don't forget to go to the SonarCloud and mark it as an intentional change (it allows doing it if you login).

Reply:
I changed the method I set up the form in the test so that code repetition is no longer an issue. Now I have a weird Codecov report.